tests/python/pants_test/backend/python/tasks/test_native_external_library_fetch.py

```diff
@@ -0,0 +1,75 @@
+# coding=utf-8
+# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from __future__ import (absolute_import, division, generators, nested_scopes, print_function,
+                        unicode_literals, with_statement)
+
+import textwrap
+import unittest
+
+from pants.backend.native.tasks.native_external_library_fetch import (ConanRequirement,
+                                                                      NativeExternalLibraryFetch)
+from pants.util.osutil import get_normalized_os_name
+
+
+class TestConanRequirement(unittest.TestCase):
+
+  def test_parse_conan_stdout_for_pkg_hash(self):
+    tc_1 = textwrap.dedent("""
+      rang/3.1.0@rang/stable: Installing package\n
+      Requirements\n    rang/3.1.0@rang/stable from 'pants-conan-remote'
+      \nPackages\n    rang/3.1.0@rang/stable:5ab84d6acfe1f23c4fae0ab88f26e3a396351ac9\n\n
+      rang/3.1.0@rang/stable: Already installed!\n"
```

Comment:
non-blocking: would be worth exploring if there is more machine friendly output from conan, and possibly using conan as a library https://github.com/conan-io/conan

Reply:
I will make a ticket to follow up with the conan team about this, and also to explore direct library usage for a later iteration.